code/style.css

```diff
@@ -1,4 +1,341 @@
-html {
-  background: #0025ff;
-  color: #fff;
+html,
+body {
+  background:whitesmoke;
+  color: black;
+  margin: 0 auto;
 }
+
+h2 {
+  font-family: 'ITC Avantgarde', serif;
+  font-size: 24px;
+  opacity: 0.8;
+}
+
+h3 {
+  font-family: 'Helvetica Neue', sans-serif;
+  font-size: 13px;
+  line-height: 1.4;
+  opacity: 0.5;
+  margin-bottom: 0;
+}
+
+p {
+  line-height: 1.4;
+  font-family: 'Helvetica Neue', sans-serif;
+  opacity: 0.8;
+}
+
+/*** HEADER ***/
+
+header {
+  background: beige;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+  font-size: 18px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  overflow: hidden;
+  z-index: 1;
+}
+
+.site-name {
+  color: black;
+  opacity: 0.8;
+  font-size: 34px;
+  font-family: 'ITC Avantgarde', serif;
+  font-weight: bolder;
+}
+
+header img {
+  width: 100px;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background:black;
+  opacity: 0.8;
+  border-radius: 50px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: black;
+  text-decoration: none;
+  opacity: 0.8;
+  padding-right: 5px;
+}
+
+/*** BANNER ***/
+
+ .banner-container {
+  display: flex;
+  height: 50px;
+  background-color: turquoise;
+  animation-duration: 0.4s;
+  animation-name: subscribe;
+  animation-iteration-count: 10;
+  animation-direction: alternate;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  width: 100%;
+  top: 100px;
+  overflow: hidden;
+  z-index: 1;
+ }
+
+ .banner-container p {
+   margin: 0;
+   font-family: 'ITC Avantgarde', serif;
+ }
+
+ @keyframes subscribe {
+   from {background-color:beige;}
+   to {background-color: turquoise;}
+ }
+
+ .banner-container p:hover {
+   animation: shake 0.5s;
+   animation-iteration-count: infinite;
+ }
+
+ @keyframes shake {
+  0% { transform: translate(1px, 1px) rotate(0deg); }
+  10% { transform: translate(-1px, -2px) rotate(-1deg); }
+  20% { transform: translate(-3px, 0px) rotate(1deg); }
+  30% { transform: translate(3px, 2px) rotate(0deg); }
+  40% { transform: translate(1px, -1px) rotate(1deg); }
+  50% { transform: translate(-1px, 2px) rotate(-1deg); }
+  60% { transform: translate(-3px, 1px) rotate(0deg); }
+  70% { transform: translate(3px, 1px) rotate(-1deg); }
+  80% { transform: translate(-1px, -1px) rotate(1deg); }
+  90% { transform: translate(1px, 2px) rotate(0deg); }
+  100% { transform: translate(1px, -2px) rotate(-1deg); }
+}
```

Comment:
I like this one though! A good thing to remember is to always declare the animation before using it. Meaning - switch the order of these two to:

```
@keyframes shake {
  0% { transform: translate(1px, 1px) rotate(0deg); }
  10% { transform: translate(-1px, -2px) rotate(-1deg); }
  20% { transform: translate(-3px, 0px) rotate(1deg); }
  30% { transform: translate(3px, 2px) rotate(0deg); }
  40% { transform: translate(1px, -1px) rotate(1deg); }
  50% { transform: translate(-1px, 2px) rotate(-1deg); }
  60% { transform: translate(-3px, 1px) rotate(0deg); }
  70% { transform: translate(3px, 1px) rotate(-1deg); }
  80% { transform: translate(-1px, -1px) rotate(1deg); }
  90% { transform: translate(1px, 2px) rotate(0deg); }
  100% { transform: translate(1px, -2px) rotate(-1deg); }
}

.banner-container p:hover {
  animation: shake 0.5s;
  animation-iteration-count: infinite;
}

```